src/config/dynamic-theme-fixes.config

```diff
@@ -11495,6 +11495,20 @@ INVERT
 
 ================================
 
+osu.edu
+ohio-state.edu
```

Comment:
Doesn't seem to be a site?

Reply:
Primarily the [login page](https://buckeyelink.osu.edu/?login=true) (redirects to it) that students and staff visit daily